Answers.md

```diff
@@ -0,0 +1,14 @@
+1. If you saw this HTML: ```<div class="box box1 box2 box3"></div>``` which class has the most specificity weight?
+Box3
+2. Describe the difference between ```display: block;``` and ```display: inline;```.
+Block takes the whole width and inline takes up what it needs.
+3. While using flexbox, what axis are you using when you use the property: ```align-items: center```?
+Cross
+4. What is the difference between fixed layout, adaptive layout, fluid layout, and responsive layout?
+Fixed -- Is only good for one screen size and will not work well on all screen sizes.
+Adaptive -- Will adjust to the user screen size.
+Fluid -- Will work somewhat time adaptive but well remove some content on smaller screens;
+Responsive -- Are better work people that have eye problems. The site will work with their font settings
+5. Why do we need to use the CSS property max-width on the outter most container in a responsive website?
+If you need have not a max width it would because problems for those of us that have larger screen like 34+ inches
```

Comment:
Good example you provided, the key word is that max-width provides a constraint~